cupyx/scipy/sparse/linalg/_iterative.py

```diff
@@ -339,7 +339,6 @@ def _make_fast_matvec(A):
     from cupy_backends.cuda.libs import cusparse as _cusparse
     from cupyx import cusparse
 
-    matvec = None
```

Comment:
`None` should be returned if `_csr.isspmatrix_csr(A) and cusparse.check_availability('spmv')` is False.

Reply:
Fixed in f7219c58432367e48ccb65bdfc8d55ed35235c50 .